### ISSUE TITLE
How to install with Docker ?

### ISSUE THREAD

**Issue by ChRiiS** [Androz2091/discord-data-package-explorer#23]
I tried to run this with Docker but can't find how...
I started with this Dockerfile :
```
FROM node
WORKDIR /usr/local/app
RUN git clone https://github.com/Androz2091/discord-data-package-explorer.git .
RUN yarn install 
EXPOSE 5000
CMD ["yarn", "dev"]
```
No problem building it, it seems to work on port 5000 so I tried to run it with a 5000:5000 redirection :
```
yarn run v1.22.5
$ rollup -c -w
rollup v1.32.1
bundles src/main.js → public/build/bundle.js...
(node:29) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /usr/local/app/node_modules/tslib/package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
[rollup-plugin-svelte] The following packages did not export their `package.json` file so we could not check the "svelte" field. If you had difficulties importing svelte components from a package, then please contact the author and ask them to export the package.json file.
- fflate
LiveReload enabled
created public/build/bundle.js in 5.7s
> discord-data-package-explorer@1.6.2 start
> sirv public "--dev"
Your application is ready~! 🚀
- Local: http://localhost:5000
LOGS
```
Everything seem fine, but connections keeps trying over and over again:
```
--2021-05-02 15:08:26--  (try: 8)  http://localhost:5000/
Connecting to localhost (localhost)|127.0.0.1|:5000... connected.
HTTP request sent, awaiting response... No data received.
Retrying.
```
So i tried to create a Dockerfile with npm:
```
FROM node
WORKDIR /usr/local/app
RUN git clone https://github.com/Androz2091/discord-data-package-explorer.git .
EXPOSE 5000
RUN npm install 
CMD ["npm", "run", "dev"]
```
Rollup module make a conflit  :
```
[4/4] RUN npm install:
#7 3.238 npm notice
#7 3.238 npm notice New minor version of npm available! 7.10.0 -> 7.11.2
#7 3.238 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.11.2>        
#7 3.238 npm notice Run `npm install -g npm@7.11.2` to update!
#7 3.238 npm notice
#7 3.244 npm ERR! code ERESOLVE
#7 3.253 npm ERR! ERESOLVE unable to resolve dependency tree
#7 3.253 npm ERR!
#7 3.254 npm ERR! While resolving: discord-data-package-explorer@1.6.2
#7 3.254 npm ERR! Found: rollup@1.32.1
#7 3.255 npm ERR! node_modules/rollup
#7 3.256 npm ERR!   dev rollup@"^1.20.0" from the root project
#7 3.256 npm ERR!
#7 3.257 npm ERR! Could not resolve dependency:
#7 3.257 npm ERR! peer rollup@">=2.0.0" from rollup-plugin-svelte@7.1.0
#7 3.258 npm ERR! node_modules/rollup-plugin-svelte
#7 3.258 npm ERR!   dev rollup-plugin-svelte@"^7.1.0" from the root project
#7 3.259 npm ERR!
#7 3.260 npm ERR! Fix the upstream dependency conflict, or retry
#7 3.260 npm ERR! this command with --force, or --legacy-peer-deps
#7 3.260 npm ERR! to accept an incorrect (and potentially broken) dependency resolution.#7 3.260 npm ERR!
#7 3.261 npm ERR! See /root/.npm/eresolve-report.txt for a full report.
#7 3.275
#7 3.275 npm ERR! A complete log of this run can be found in:
#7 3.275 npm ERR!     /root/.npm/_logs/2021-05-02T13_10_45_845Z-debug.log
------
executor failed running [/bin/sh -c npm install]: exit code: 1
```
So I tried with --force (and with --legacy-peer-deps another time), it could be built and started
``` 
> discord-data-package-explorer@1.6.2 dev
> rollup -c -w
rollup v1.32.1
bundles src/main.js → public/build/bundle.js...
(node:19) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /usr/local/app/node_modules/tslib/package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
[rollup-plugin-svelte] The following packages did not export their `package.json` file so we could not check the "svelte" field. If you had difficulties importing svelte components from a package, then please contact the author and ask them to export the package.json file.
- fflate
LiveReload enabled
created public/build/bundle.js in 6.1s
> discord-data-package-explorer@1.6.2 start
> sirv public "--dev"
Your application is ready~! 🚀
- Local: http://localhost:5000
LOGS
```
But again as with yarn, no answer ... Any clue ?

**Comment by ChRiiS** [comment]
Just in case, I just tried with an old node:14 source, it can be build and run but there's a "Chart" error : 
```
> discord-data-package-explorer@1.6.2 dev /usr/local/app
> rollup -c -w
rollup v1.32.1
bundles src/main.js → public/build/bundle.js...
[!] Error: 'Chart' is not exported by node_modules/frappe-charts/dist/frappe-charts.min.umd.js, imported by node_modules/svelte-frappe-charts/src/components/base.svelte
https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module
node_modules/svelte-frappe-charts/src/components/base.svelte (3:11)
1: <script>
2: import { onMount, afterUpdate, onDestroy } from 'svelte';
3: import { Chart } from 'frappe-charts';
^
4:
5: /**
Error: 'Chart' is not exported by node_modules/frappe-charts/dist/frappe-charts.min.umd.js, imported by node_modules/svelte-frappe-charts/src/components/base.svelte
at error (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:5400:30)
at Module.error (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:9824:16)
at handleMissingExport (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:9725:28)
at Module.traceVariable (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:10163:24)
at ModuleScope.findVariable (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:8770:39)
at FunctionScope.findVariable (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:3065:38)
at ChildScope.findVariable (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:3065:38)
at ReturnValueScope.findVariable (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:3065:38)
at ChildScope.findVariable (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:3065:38)
at Identifier$1.bind (/usr/local/app/node_modules/rollup/dist/shared/node-entry.js:4403:40)
```
And still no HTTP response...

**Comment by Androz2091** [comment]
This is now fixed in v2.1.0
